website/src/theme/BoxOut/styles.module.css

```
@@ -8,8 +8,16 @@
     display: flex;
     padding-bottom: 1rem;
     align-items: center;
-    margin: 0;
-    padding: 2rem 2rem 1rem 2rem;
+    margin: .3rem;
```

```suggestion
    margin: 0.3rem;
```
Its a nitpick, but a `0` before a `.x` adds to readability.